From: <hesso@pool.math.tu-berlin.de>
Subject: Remove the calls to glib-config
 glib-config is a tool from libglib1.2 and therefore the wrong one to
 ask) and replace them with the ./configure macros that have already
 been collected.

--- gdome2-0.8.1/gdome-config.in.orig	2007-07-07 00:15:30.000000000 +0200
+++ gdome2-0.8.1/gdome-config.in	2007-07-07 00:14:15.000000000 +0200
@@ -83,8 +83,8 @@
     shift
 done
 
-the_libs="$the_libs -L${exec_prefix}/lib -lgdome `@GLIB_CONFIG@ --libs` `@XML_CONFIG@ --libs`"
-the_flags="$the_flags -I${prefix}/include -I${prefix}/include/libgdome `@GLIB_CONFIG@ --cflags` `@XML_CONFIG@ --cflags`"
+the_libs="$the_libs -L${exec_prefix}/lib -lgdome @GLIB_LIBS@ `@XML_CONFIG@ --libs`"
+the_flags="$the_flags -I${prefix}/include -I${prefix}/include/libgdome @GLIB_CFLAGS@ `@XML_CONFIG@ --cflags`"
 
 if $cflags; then
     all_flags="$the_flags"
